multi-agent/lib/mas/elements/nodes/claude_agent/claude_agent_node.py

```
@@ -263,6 +275,60 @@ async def _async_execute(
+                    #             ),
+                    #             "call_id": block.tool_use_id,
+                    #             "output": self._extract_tool_result_text(
+                    #                 block.content
+                    #             ),
+                    #         })
+
+            elif isinstance(message, UserMessage):
+                if isinstance(message.content, list):
+                    for block in message.content:
+                        if isinstance(block, ToolResultBlock):
+                            if self.is_streaming():
```

Ruff (0.15.15)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)